packages/patternfly-4/react-core/src/components/ChipGroup/ChipGroupToolbarItem.tsx

```diff
@@ -10,25 +13,52 @@ export interface ChipGroupToolbarItemProps extends React.HTMLProps<HTMLUListElem
   children: React.ReactNode;
   /** Additional classes added to the chip item */
   className?: string;
+  /** Flag if chip group can be closed*/
+  isClosable?: boolean;
+  /** Function that is called when clicking on the chip group button */
+  onClick?: (event: React.MouseEvent) => void;
+  /** Aria label for close button */
+  closeBtnAriaLabel?: string;
 }
 
 export const ChipGroupToolbarItem: React.FunctionComponent<ChipGroupToolbarItemProps> = ({
   categoryName = '',
   children = null,
   className = '',
+  isClosable = false,
+  onClick = (_e: React.MouseEvent) => undefined as any,
+  closeBtnAriaLabel = 'close chip group',
```

Comment:
Not sure how important this is, but we typically sentence case things.

```suggestion
  closeBtnAriaLabel = 'Close chip group',
```

Reply:
@nicolethoen LGTM after update to sentence case 👍